dotcom-rendering/src/web/components/LeftColumn.tsx

```diff
@@ -64,41 +64,34 @@ const partialRightBorder = (colour: string) => css`
 	}
 `;
 
-const rightBorder = (colour: string) => css`
+const fullRightBorder = (colour: string) => css`
 	border-right: 1px solid ${colour};
 `;
 
 type Props = {
 	children: React.ReactNode;
-	showRightBorder?: boolean;
-	showPartialRightBorder?: boolean;
+	borderType?: 'full' | 'partial';
 	borderColour?: string;
 	size?: LeftColSize;
 };
 
 export const LeftColumn = ({
 	children,
-	showRightBorder = true,
```

Comment:
I noticed there were a number of chromatic diffs where the right border is now missing. We might want to consider setting a default value for `borderType` too? E.g. ArticleHeadline uses left column like:

```
			<Flex>
				<LeftColumn>
					<></>
				</LeftColumn>
```

Previously, if neither `showRightBorder` or `showPartialRightBorder` were defined then I think we defaulted to showing the full right border:

```
If props not defined:
showRightBorder = true
showPartialRightBorder = false
shouldShowPartialBoard = false

--> border styling applied to <section>
```

At the moment, if `borderType` is not defined, I think we get no right border at all, so the default behaviour has changed (which is why we're seeing diffs for things like ArticleHeadline)? Or I'm confusing myself (or this change is intended)!

Reply:
Hmm, it was confusing to understand the default before because there were two booleans... but you're absolutely right, we should keep to that api otherwise we risk subtle regressions. I'll have a look at this